docker/vsi_common/makeself.Justfile

```diff
@@ -105,7 +111,7 @@ function caseify()
         extra_args+=1
       fi
 
-      # Start by adding just vsi_common, it will be transformed (in makefile_prep)
+      # Start by adding only vsi_common, it will have been transformed (by makeself_git_prep)
```

Comment:
I prefer `it will be`. If you say `will have been`, it seems like it should be `will have been transformed (by makeself_git_prep) by the time [such and such happens]`

Reply:
It looks like makeself_git_prep was just run. Did you mean, `it has been transformed`?

Reply:
The transformation hasn't happened until after makeself is called, and it happens DURING the makeself call. The calculation has been made by `makeself_git_prep`, but the transformation has yet to come. It hasn't actually "happened" happened yet, it has "will have going to have happened" happened

```suggestion
      # Start by adding only vsi_common; it will be transformed (due to makeself_git_prep)
```